src/App.js

```diff
@@ -1,25 +1,24 @@
-import logo from './logo.svg';
 import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import Header from './components/Header';
+import Nav from './components/Nav';
+import Articles from './components/Articles';
+import Topics from './components/Topics';
```

Comment:
When I load your app up, there is an error at the moment with the import of your topics, from the Component directory. This seems to be due to their not being a Topics component in your components directory.